Field-Robotics-Lab/dave
# Predator fix

Fixes issue #48 by adding back the texture tags. This only works in Gazebo 9. Issue #97 has been created to address Gazebo 11 compatibility. This also does not fix inertia values (see issue #98). All it fixes is the texture.

To test:

```
roslaunch uuv_dave uuv_predator.launch
```
Verify that the predator has the correct (black texture) and is not the light gray material that appears when the texture isn't loading. That is, it should look like this:

![predator_reborn](https://user-images.githubusercontent.com/8611855/108930261-762f0980-75fa-11eb-90c4-503c54601c96.png)

Not this:
![predator_no_texture](https://user-images.githubusercontent.com/8611855/108930365-aaa2c580-75fa-11eb-9afd-de7284879b21.png)

For thoroughness, one might also check that the controls work:

Right Joystick
|Key |Action|
|-----------|------------|
| Down | Shoulder joint lowers |
| Up | Shoulder joint raises |
|Left | Shoulder joint swivel left |
|Right| Shoulder joint swivel right |

Left Joystick
|Key |Action|
|-----------|------------|
| Down | Elbow joint lowers |
| Up | Elbow joint raises |
|Left | Wrist joint swivel left |
|Right| Wrist joint swivel right |

4-way keypad
|Key |Action|
|-----------|------------|
| Down | Wrist joint lowers |
| Up | Wrist joint raises |

XYAB Buttons
|Key |Action|
|-----------|------------|
| B | Claw opens |
|X | Claw closes |
